test/T80.custom_trampoline.include

```diff
@@ -0,0 +1,6 @@
+#include<string>
+
+template <typename A, typename trampoline_A>
+A myFoo(const trampoline_A *ptr_to_this, const std::string &class_name, const std::string &function_name) { 
+    return A();
+};
```

Comment:
somewhat irrelevant now but i want to mention it in case you face similar task in the future: for testing we can simply  declare this function in test/T80.custom_trampoline.hpp file: since it is templated it will be ignored by Binder anyway.